tiatoolbox/models/engine/nucleus_instance_segmentor.py

```diff
@@ -655,6 +655,10 @@ def _predict_one_wsi(
                     index_by_id[id(geo)] for geo in spatial_indexer.query(sel_box)
                 ]
 
+                # there is nothing in the tile
+                if len(sel_indices) == 0:
+                    continue
```

Comment:
Please fix the codecov issue at this line and we are good to merge.

Reply:
At the moment, this was detected on the CMU-small.svs which is around 3k x 3k. Replacing the test sample with it will include this in coverage but that means longer travis running time. I think we should ignore this travis coverage as I can include this sample for checking with the local tests.

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/257/commits/cf6f6a72c295867abf44c0a3231efbe3dfe03fe3